scalableminds/webknossos
# ignoring scrolling input when dragging on a plane

This PR disables scrolling while dragging a plane.

### URL of deployed dev instance (used for testing):
- [https://dontscrollwhiledragging.webknossos.xyz](https://dontscrollwhiledragging.webknossos.xyz)

### Steps to test:
- Open some tracing.
- Scrolling should not work while dragging a plane.
- Scrolling should work when not dragging.
- Dragging so far that the mouse enters another plane and then scrolling still should disable scrolling.

### Issues:
- fixes #3933

------

#### TODO
- [x] Handle the 4th behavior mentioned in steps to test.
